templates/common/_base/units/wait-for-primary-ip.yaml

```diff
@@ -9,9 +9,15 @@ contents: |
 
   [Service]
   Type=oneshot
-  Restart=on-failure
-  RestartSec=10
-  ExecStart=/usr/local/bin/wait-for-primary-ip.sh
+  # Would prefer to do Restart=on-failure instead of this bash retry loop, but
+  # the version of systemd we have right now doesn't support it. It should be
+  # available in systemd v244 and higher.
+  ExecStart=/bin/bash -c " \
```

Comment:
Ah, it's complaining that this doesn't seem to be syntactically valid:

```
Sep 20 09:26:30 master-0.ostest.test.metalkube.org bash[2445]: /bin/bash: -c: line 1: syntax error near unexpected token `done'
Sep 20 09:26:30 master-0.ostest.test.metalkube.org bash[2445]: /bin/bash: -c: line 1: `    until    /usr/local/bin/resolv-prepender.sh    do    sleep 10;    done'
Sep 20 09:26:30 master-0.ostest.test.metalkube.org systemd[1]: on-prem-resolv-prepender.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Sep 20 09:26:30 master-0.ostest.test.metalkube.org systemd[1]: on-prem-resolv-prepender.service: Failed with result 'exit-code'.
```

looking at the failed CI runs

Reply:
Missing semicolon in the bash loop. Fixed